test/functional/test-storage.js

```diff
@@ -502,6 +503,17 @@ describe('LocalStorageBinding', () => {
         });
   });
 
+  it('should bypass loading from localStorage if getItem throws', () => {
+    localStorageMock.expects('getItem')
+        .throws(new Error('unknown'))
+        .once();
+    binding = new LocalStorageBinding(windowApi);
+    return binding.loadBlob('https://acme.com')
```

Comment:
I think we need another expects here, to show that we aren't using localstorage anymore.